frontend/src/components/AboutDialog/AboutDialog.tsx

```diff
@@ -13,8 +13,8 @@ type Props = {
 export let AboutDialog: React.StatelessComponent<Props> = ({
   isOpen, onCloseClick
 }) => {
-  let bayesLink = SecureLink('https://www.bayesimpact.org', 'Bayes Impact')
-  let emailLink = SecureLink('mailto:encompass@bayesimpact.org?subject=About%20Encompass', 'encompass@bayesimpact.org')
+  let beaconLink = SecureLink('https://www.thebeaconlabs.org', 'Beacon Labs')
+  let emailLink = SecureLink('mailto:encompass@thebeaconlabs.org?subject=About%20Encompass', 'encompass@thebeaconlabs.org')
   let githubLink = SecureLink('https://github.com/bayesimpact/encompass', 'GitHub repository')
```

Comment:
change GitHub link?

Reply:
do we have a new github org? that's one of the things remaining